packages/core/src/model.ts

```diff
@@ -52,18 +52,29 @@ export class ChartModel {
 		const { groupMapsTo } = this.getOptions().data;
 
 		const axesOptions = this.getOptions().axes;
-		
+
 		// Check for custom domain
 		if (axesOptions) {
 			Object.keys(axesOptions).forEach(axis => {
-				if (axesOptions[axis].domain && axesOptions[axis].mapsTo) {
-					const mapsTo = axesOptions[axis].mapsTo;
-					const [start, end] = axesOptions[axis].domain;
-
-					// Filter out data outside domain
-					displayData = displayData.filter((datum) => 
-						datum[mapsTo] >= start && datum[mapsTo] <= end
-					)
+				const mapsTo = axesOptions[axis].mapsTo;
+
+				if (
```

Comment:
Before it checked for the existence of `axesOptions[axis].mapsTo` before setting it to mapsTo. Do you think it makes more sense to do that because if mapsTo doesn't exist it won't be used anywhere else?

Reply:
I know, but `continue` doesn't work in `forEach`, I need to turn `forEach` to a `for` loop for that.

Reply:
If the `axesOptions[axis].mapsTo` doesn't exist, then the `const mapsTo` will be undefined. In the `if` check, if mapsTo is undefined, it will skip the rest and won't use `mapsTo` in anywhere.

Reply:
So you can't do something like:
```
	if (axesOptions[axis].mapsTo
		&& axesOptions[axis].domain
		&& !this.getOptions().percentage){
		mapsTo = axesOptions[axis].mapsTo;
        ...etc
	}
```
?